docs/content/error/$injector/modulerr.ngdoc

```diff
@@ -15,7 +15,7 @@ In AngularJS `1.2.0` and later, `ngRoute` has been moved to its own module.
 If you are getting this error after upgrading to `1.2.x` or later, be sure that you've
 installed {@link ngRoute `ngRoute`}.
 
-### Monkey-patching Angular's `ng` module
+### Monkey-patching AngularJS's `ng` module
```

Comment:
Hm...according to [this][1] it's fine.

[1]: http://data.grammarbook.com/blog/apostrophes/apostrophes-with-words-ending-in-s/